tests/Unit/DataStructures/Tensor/Expressions/Test_EvaluateComplex.cpp

```diff
@@ -386,60 +409,68 @@ void test_evaluate_large_expressions(
+  CHECK_ITERABLE_APPROX(get(complex_tensor_times_complex_tensor_result),
                         64.0 * get(complex_scalar));
-  CHECK_ITERABLE_APPROX(get(complex_tensor_plus_real_tensor_result),
-                        64.0 * (get(complex_scalar) + get(real_scalar)));
-  CHECK_ITERABLE_APPROX(get(real_tensor_plus_complex_tensor_result),
-                        64.0 * (get(complex_scalar) + get(real_scalar)));
+  CHECK_ITERABLE_APPROX(get(complex_tensor_times_real_tensor_result),
+                        64.0 * 64.0 * (get(complex_scalar) * get(real_scalar)));
+  CHECK_ITERABLE_APPROX(get(real_tensor_times_complex_tensor_result),
+                        64.0 * 64.0 * (get(complex_scalar) * get(real_scalar)));
 
   // check expressions with `Tensor`s and numbers
-  CHECK_ITERABLE_APPROX(get(real_tensor_plus_real_number_result),
-                        64.0 * get(real_scalar) + real_number);
-  CHECK_ITERABLE_APPROX(get(real_number_plus_real_tensor_result),
-                        64.0 * get(real_scalar) + real_number);
-  CHECK_ITERABLE_APPROX(get(complex_tensor_plus_real_number_result),
-                        64.0 * get(complex_scalar) + real_number);
-  CHECK_ITERABLE_APPROX(get(real_number_plus_complex_tensor_result),
-                        64.0 * get(complex_scalar) + real_number);
-  CHECK_ITERABLE_APPROX(get(complex_tensor_plus_complex_number_result),
-                        64.0 * get(complex_scalar) + complex_number);
-  CHECK_ITERABLE_APPROX(get(complex_number_plus_complex_tensor_result),
-                        64.0 * get(complex_scalar) + complex_number);
+  CHECK_ITERABLE_APPROX(get(real_tensor_times_real_number_result),
+                        64.0 * get(real_scalar) * real_number);
+  CHECK_ITERABLE_APPROX(get(real_number_times_real_tensor_result),
+                        64.0 * get(real_scalar) * real_number);
+  CHECK_ITERABLE_APPROX(get(complex_tensor_times_real_number_result),
+                        64.0 * get(complex_scalar) * real_number);
+  CHECK_ITERABLE_APPROX(get(real_number_times_complex_tensor_result),
+                        64.0 * get(complex_scalar) * real_number);
+  CHECK_ITERABLE_APPROX(get(real_tensor_times_complex_number_result),
+                        64.0 * get(real_scalar) * complex_number);
+  CHECK_ITERABLE_APPROX(get(complex_number_times_real_tensor_result),
+                        64.0 * get(real_scalar) * complex_number);
+  CHECK_ITERABLE_APPROX(get(complex_tensor_times_complex_number_result),
+                        64.0 * get(complex_scalar) * complex_number);
+  CHECK_ITERABLE_APPROX(get(complex_number_times_complex_tensor_result),
+                        64.0 * get(complex_scalar) * complex_number);
 }
 }  // namespace
 
 SPECTRE_TEST_CASE("Unit.DataStructures.Tensor.Expression.EvaluateComplex",
                   "[Unit][DataStructures]") {
   MAKE_GENERATOR(generator);
 
-  const size_t vector_size = 3;
+  const size_t vector_size = 2;
```

Comment:
Why the size change?

Reply:
I just figured there were a lot of FLOPs happening in this test and this just made it slightly less, but there is not reason it can't still be 3

Reply:
That's fine, was just curious.